pallets/parachain-staking/src/lib.rs

```diff
@@ -1727,14 +1727,15 @@ pub mod pallet {
 		///
 		/// The intended bond amounts will be used while calculating rewards.
 		fn get_rewardable_delegators(
+			mut total_counted: BalanceOf<T>,
```

Comment:
It's not readable, please remove this parameter and return the sum of all "pending leave" amounts and compute the subtraction in the caller.